gui/wxpython/gis_set.py

```diff
@@ -507,6 +508,10 @@ def SuggestDatabase(self):
         if self.GetRCValue("LOCATION_NAME") != "<UNKNOWN>":
             return
         path = get_possible_database_path()
+        # If nothing found, create GRASS directory
```

Comment:
Based on what the function does and what the context is, the comment should be "...try to create..."